plugins/PrivacyManager/vue/src/ManageGdpr/ManageGdpr.vue

```diff
@@ -359,6 +376,27 @@ export default defineComponent({
     };
   },
   methods: {
+    changeSite(newValue: SiteRef) {
+      AjaxHelper.fetch(
+        {
+          module: 'API',
+          method: 'Live.isVisitorProfileEnabled',
+          filter_limit: -1,
+          idSite: newValue.id,
+        },
+        {
+          createErrorNotification: false, // don't show errors from this API in UI
+        },
+      ).then((response) => {
+        if (!response.value && this.segment_filter === 'userId=') {
```

Comment:
I think the check is incorrect an causes `userid` to remain in the segment selector even if it's not available.
Can be reproduce when you switch to a single site and then back to "all".
```suggestion
        if (!response.value && this.segment_filter === 'userId==') {
```

Reply:
This was a typo, thanks for catching it.